nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EvaluateXPath.java

```diff
@@ -137,7 +138,7 @@ public class EvaluateXPath extends AbstractProcessor {
             .description("Specifies whether or not the XML content should be validated against the DTD.")
             .required(true)
             .allowableValues("true", "false")
-            .defaultValue("true")
```

Comment:
Is there a little context for this change?

Reply:
As mentioned for `EvaluateXQuery`, disabling validation of the Document Type Declaration in the default configuration provides a more secure starting point for new instances of the Processor.